src/terraform.ts

```diff
@@ -0,0 +1,228 @@
+import TelemetryReporter from '@vscode/extension-telemetry';
+import * as vscode from 'vscode';
+import { ExecuteCommandParams, ExecuteCommandRequest, LanguageClient } from 'vscode-languageclient/node';
+import { Utils } from 'vscode-uri';
+import { getActiveTextEditor } from './utils/vscode';
+import { clientSupportsCommand } from './utils/clientHelpers';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+export interface ModuleCaller {
+  uri: string;
+}
+
+export interface ModuleCallersResponse {
+  version: number;
+  moduleCallers: ModuleCaller[];
```

Comment:
While working on the mocha stuff yesterday, I noticed that the fields in the response might have the wrong names (also wrong in previous versions). Shouldn't they be called `v` and `callers` or am I missing something?

https://github.com/hashicorp/terraform-ls/blob/4e96ebe79a350c56774cb1ee1f18924ee6563f1a/internal/langserver/handlers/command/module_callers.go#L15-L17

Reply:
I think you're right. I updated the interface